server/socketAPI.js

```diff
@@ -0,0 +1,30 @@
+const socket_io = require("socket.io");
```

Comment:
This isn't in the package.json, can you npm install it? Also this variable should be updated to camelCase

Reply:
Yes, I'm not sure why it didn't get included in the package.json when I npm installed it. I think I may have accidentally overwritten it during merging.